hack/update-config.sh

```diff
@@ -22,7 +22,7 @@ TESTINFRA_ROOT=$(git rev-parse --show-toplevel)
 bazel run //config/jobs/kubernetes-security:genjobs -- \
 "--config=${TESTINFRA_ROOT}/prow/config.yaml" \
 "--config-json=${TESTINFRA_ROOT}/jobs/config.json" \
-"--jobs=${TESTINFRA_ROOT}/jobs" \
+"--jobs=${TESTINFRA_ROOT}/config/jobs" \
```

Comment:
is that the right directory? or should it be `config/jobs/kubernetes`?

Reply:
it's passing this along to the prow config loading logic to load the jobs, for which config/jobs has all the prow jobs. it would work with config/jobs/kubernetes but be inconsistent with the rest of our config loading.

Reply:
and if for some reason a kubernetes presubmit wasn't in config/jobs/kubernetes it would miss generating a security repo equivilant